python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -95,6 +95,7 @@ def __init__(
             local_pipeline_package_path, 'local_pipeline_package_path'
         )
         self.staging_prefix = check.str_param(staging_prefix, 'staging_prefix')
+        check.invariant(staging_prefix.startswith('/'), 'staging_prefix must be an absolute path')
```

Comment:
i guess `os.path.isabs` is not an option because the windows behavior would be wrong

Reply:
Yep, Databricks needs unix-style paths.

Reply:
it miiight be worth expanding the error message here to make that totally clear